workflows/toast_so_sim.py

```diff
@@ -789,7 +821,7 @@ def main():
             name="filterbin",
             enabled=False,
         ),
-        so_ops.MLMapmaker(name="mlmapmaker", enabled=False, comps="TQU"),
+        #so_ops.MLMapmaker(name="mlmapmaker", enabled=False, comps="TQU"),
```

Comment:
The MLMapmaker is disabled by default here (`enabled=False`), so there is no need to comment it out.

Reply:
Done